aepsych/utils.py

```diff
@@ -59,11 +59,11 @@ def dim_grid(
 
     for i in range(dim):
         if i in slice_dims.keys():
-            mesh_vals.append(slice(slice_dims[i] - 1e-10, slice_dims[i] + 1e-10, 1))
+            mesh_vals.append(torch.tensor([slice_dims[i]]))
```

Comment:
I believe the epsilon (`1e-10`) is needed for some use-cases, please maintain that.